scripts/run-swift-format.sh

```diff
@@ -24,6 +24,7 @@ REPO_ROOT="$(git -C "${CURRENT_SCRIPT_DIR}" rev-parse --show-toplevel)"
 SWIFTFORMAT_BIN=${SWIFTFORMAT_BIN:-$(command -v swift-format)} || fatal "❌ SWIFTFORMAT_BIN unset and no swift-format on PATH"
 
 git -C "${REPO_ROOT}" ls-files -z '*.swift' \
+  | grep -z -v 'Tests/OpenAPIGeneratorReferenceTests/Resources/ReferenceSources/Petstore' \
```

Comment:
```suggestion
  | grep -z -v 'Tests/OpenAPIGeneratorReferenceTests/Resources' \
```

And please also exclude and remove any changes to the Sources/swift-openapi-generator/Documentation.docc. Otherwise will be ready to go.